cmd/yace/main.go

```diff
@@ -167,10 +166,10 @@ func (s *scraper) decoupled(ctx context.Context, cache exporter.SessionCache) {
 	}
 }
 
-func (s *scraper) scrape(ctx context.Context, cache exporter.SessionCache) (err error) {
+func (s *scraper) scrape(ctx context.Context, cache exporter.SessionCache) {
```

Comment:
The linter was complaining about unchecked return value. For this method as it is now there's not much we can do in case of error, so I'm just removing it from signature.